templates/ai/what-is-kubeflow.html

```diff
@@ -512,14 +514,14 @@ <h2 class="u-fixed-width u-sv1">Who uses Kubeflow?</h2>
     </div>
     <div class="col-8">
       <h2 class="u-fixed-width u-sv1">Get started today</h2>
-      <p class="u-sv2 p-heading--4">Try-out Kubeflow on your K8s deployment. Or on MicroK8s - Zero-ops Kubernetes with high availability.</p>
-      <p>Single-command deploy locally on your desktop, public cloud VM or on-prem server.</p>
+      <p class="u-sv2 p-heading--4">Try-out Kubeflow on your Kubernetes environment. Or on MicroK8s - Zero-ops Kubernetes with high availability.</p>
+      <p>Deploy locally on your workstation cloud VM or on-prem server.</p>
       <p>
-        <a class="p-button--positive p-link--external" href="http://charmed-kubeflow.io/docs/install">
-        Try Kubeflow on any K8s
+        <a class="p-button--positive" href="https://charmed-kubeflow.io/docs/quickstart">
+        Try Kubeflow on MicroK8s
         </a>
-        <a class="p-link--external" href="https://microk8s.io/docs/addon-kubeflow">
-          Kubeflow on MicroK8s
+        <a href="https://charmed-kubeflow.io/#get-in-touch">
+          Contact us now
```

Comment:
There is no `>` on the copy doc but maybe it's needed here as well? (same as line 20?)   
```suggestion
        <a href="https://charmed-kubeflow.io/#get-in-touch">
          Contact us&nbsp;&rsaquo;
```